katas/content/qec_shor/index.md

```diff
@@ -17,48 +17,48 @@ This kata introduces you to the basic concepts of quantum error correction using
 
 **What you should know to start working on this kata:**
 
-- Basic single-qubit and multi-qubit gates
-- Single-qubit and multi-qubit quantum measurements and their effect on quantum systems
+- Basic knowledge of single-qubit and multi-qubit gates
+- Basic knowledge of single-qubit and multi-qubit quantum measurements and their effect on quantum systems
 
 @[section]({
     "id": "qec_shor__noise",
     "title": "Noise in Classical and Quantum Systems"
 })
 
-Any quantum system we can use to carry out quantum computation is inherently noisy. 
-Quantum noise can be caused by different physical processes, depending on the type of a particle or device used as a qubit.
-From the computation point of view, the presence of noise in the quantum system means that its state can suffer from random errors, and thus end up different from the state we're relying on to do our computations. 
+Any quantum system that can carry out quantum computation is inherently noisy. 
```

Comment:
Please revert this line change. Quantum systems don't carry out computations themselves, we use them to do that
```suggestion
Any quantum system we can use to carry out quantum computation is inherently noisy. 
```

Reply:
Neat observation